assets/queries/cloudFormation/user_data_base64_contains_encoded_private_key/test/positive.json

```diff
@@ -0,0 +1,16 @@
+{
+  "Resources":{
+    "myLaunchConfig":{
```

Comment:
Change "myLaunchConfig" to "myLaunchConfig3"

Reply:
done